README.md

```diff
@@ -19,6 +19,16 @@
 
 ## General Setup and Install
 
+### Using shield CLI
+The [`shield` CLI tool](https://www.npmjs.com/package/shield-study-cli) makes it easy to start a Shield study with this add-on template.
+
+1. Install `shield`:
+  * `npm install shield-study-cli`
```

Comment:
Not sure how/if this will work as-is.

I think it'd either have to be:

0. `npm install shield-study-cli -g` to install globally (and be able to refer to it as `shield` below), or should be 
0. `./node_modules/.bin/shield init <change>-shield-study` below to use the locally installed **shield-study-cli** module.